examples/experimental/trial/bert_squad_pytorch/const.yaml

```diff
@@ -8,19 +8,26 @@ hyperparameters:
     adam_epsilon: 1e-8
     weight_decay: 0
     num_warmup_steps: 0
-    num_training_steps: 400
     max_seq_length: 384
     doc_stride: 128
     max_query_length: 64
     n_best_size: 20
     max_answer_length: 30
     null_score_diff_threshold: 0.0
+    max_grad_norm: 1.0
+    num_training_steps: 15000 # This is the number of optimizer steps. Set it
```

Comment:
this is the main fix to achieve parity.  if this value is too low, [we end up with a LR of zero](https://github.com/huggingface/transformers/blob/eacea530c1ed63b182433a1b462d33881bd20409/src/transformers/optimization.py#L55-L57) pretty quickly, and this was causing the learning to basically stop (validation f1 flatlined at 60-something).

in the huggingface repo, they set this similarly to # batches times * epochs (assuming gradient accumulation is 1) -- see [here](https://github.com/huggingface/transformers/blob/master/examples/question-answering/run_squad.py#L87) and [here](https://github.com/huggingface/transformers/blob/master/examples/question-answering/run_squad.py#L100).

with this fix we get closer to the expected validation f1: 89 after 150 steps, compared to 88.52.